lib/plugins/create/templates/aws-nodejs/handler.js

```diff
@@ -1,8 +1,24 @@
 'use strict';
 
 // Your first function handler
-module.exports.hello = (event, context, cb) => {
-  cb(null, { message: 'Go Serverless v1.0! Your function executed successfully!', event });
-};
+module.exports.hello = (event, context, callback) => {
+  // This code is used so that your function can repond to HTTP events
+  // which use the LAMBDA-PROXY integration
+  const body = {
+    message: 'Go Serverless v1.0! Your function executed successfully!',
+    input: event,
+  };
+
+  const response = {
+    statusCode: 200,
+    headers: {
+      'custom-header': 'Custom header value',
+    },
+    body: JSON.stringify(body),
```

Comment:
why do we `JSON.stringify` here? isn't that handled by AWS? if not, then shouldn't we stringify the entire `response` object?